src/Scaffolding/VS.Web.CG.Mvc/Minimal Api/MinimalApiGenerator.cs

```diff
@@ -179,11 +181,31 @@ internal async Task AddEndpointsMethod(string membersBlockText, string endpoints
 
                     if (classNode != null && classNode is ClassDeclarationSyntax classDeclaration)
                     {
+                        SyntaxNode classParentSyntax = null;
+                        //if class is not static, create a new class in the same file
+                        if (!classDeclaration.Modifiers.Any(x => x.Text.Equals(SyntaxFactory.Token(SyntaxKind.StaticKeyword).Text)))
```

Comment:
Can x.Text be null?

Reply:
No, it can string.Empty but can't be null.